File: lets_invest/lib/data/Crypto.dart
class Crypto {
  String name;
  String isin;
  dynamic bid;
  double quantity;
  double boughtAT;
  Crypto(
      {required this.name,
      required this.isin,
      required this.bid,
      required this.quantity,
      required this.boughtAT});
}

File: lets_invest_api/lets_invest_api/Models/Crypto.cs
namespace lets_invest_api.Models
{
    public class Crypto
    {
        public int Id { get; set; }
        
        public string Name { get; set; }
        
        public string Isin { get; set; }

        public double Quantity { get; set; }

        public DateTime BoughtAt { get; set; } = DateTime.UtcNow;
    }
}
